store/serializers.py

```diff
@@ -461,7 +461,11 @@ def create(self, validated_data):
                     ]
                     coupon_info['orderline'].save()
                 else:
-                    raise serializers.ValidationError(coupon_info['error'])
+                    raise serializers.ValidationError({
+                        "non_field_errors": [
+                            coupon_info['error']
+                        ],
+                    })
```

Comment:
Not related to this PR, please remove this

Reply:
done